alerts/cluster-network-operator/NoRunningOvnMaster.md

```diff
@@ -0,0 +1,74 @@
+# NoRunningOvnMaster
+
+## Meaning
+
+This [alert][NoRunningOvnMaster] is triggered when there are no
+OVN-Kubernetes master control plane pods
+[Running][PodRunning]
+This is a critical-level alert if no OVN-Kubernetes master control plane pods
+are not running for `10m`.
+
+## Impact
+Networking control plane is not functional. Networking configuration updates
+will not be applied to the cluster.
+Without a functional networking control plane, existing workloads may continue
+to be partially functional,
+but new workloads will not be functional.
+Updates required for functioning Kubernetes services will not be performed.
+
+## Diagnosis
+### Cluster network operator
+Cluster Network operator (CNO) manages the CRUD operations for OVN-Kubernetes
+daemonset.
+Verify the CNO is running:
+
+    oc -n openshift-network-operator get pods -l name=network-operator
+
+Verify the [CNO](https://github.com/openshift/cluster-network-operator/) is
+functioning without error by checking the operator Status.
+You can check the Status of the CNO directly using the following command:
```

Comment:
Maybe first check if the status is ok with `oc get co network` and only if not go further with CNO?